cli/Valet/DevTools.php

```diff
@@ -51,17 +48,35 @@ function __construct(Brew $brew, CommandLine $cli, Filesystem $files,
      *
      * @return void
      */
-    function install()
+    function install($skipTools)
     {
         info('[devtools] Installing tools');
 
-        foreach (self::SUPPORTED_TOOLS as $tool) {
+        foreach (self::BREW_SUPPORTED_TOOLS as $tool) {
+            if (in_array($tool, $skipTools)) {
+                continue;
+            }
             if ($this->brew->installed($tool)) {
                 info('[devtools] ' . $tool . ' already installed');
             } else {
                 $this->brew->ensureInstalled($tool, []);
             }
         }
+
+        if (!in_array('wp-cli', $skipTools)) {
+            info('Installing wp-cli...');
+            if (exec('wp --info')) {
+                info('wp-cli is already installed');
+            } else {
+                echo shell_exec(
```

Comment:
I wouldn't echo here, use `info()` for example

Reply:
Alright